### PR TITLE
Update KERAS OCR TFLite Model

### PR DESCRIPTION
**Pull request by tulasiram58827**
Updating TFLite Model which also includes post-processing steps(CTC Decoder)

Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms.

### CODE REVIEW

**Comment by sayakpaul** [comment]
@tulasiram58827 what's the problem here?

**Comment by tulasiram58827** [comment]
@sayakpaul  I don't know actually I just changed the model file path. I did not add any extra lines, not even a word. 

This is the validation error I can see in the log.

> __main__.MarkdownDocumentationError: Error at file /home/runner/work/tfhub.dev/tfhub.dev/assets/docs/tulasiram58827/keras-ocr/dr/1.md: Unexpected line found: "[![Open Colab Notebook]](https://colab.research.google.com/github/tulasiram58827/ocr_tflite/blob/main/colabs/KERAS_OCR_TFLITE.ipynb)". Please refer to [README.md](https://github.com/tensorflow/hub/blob/master/tensorflow_hub/tfhub_dev/README.md) for information about markdown format.

Can you interpret anything from here? File format is the same as other files I committed

**Comment by sayakpaul** [comment]
It should be "Open Colab notebook".

**Comment by tulasiram58827** [comment]
Oh my bad. Thanks @sayakpaul

Update.

File: assets/docs/tulasiram58827/keras-ocr/dr/1.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # Lite tulasiram58827/keras-ocr/dr/1
-
 TF Lite quantized version of the Keras OCR Model for text recognition from images.
 
 <!-- parent-model: tulasiram58827/keras-ocr/1 -->
-<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.1/keras_ocr_dr.tar.xz -->
+<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.2/keras_ocr_dr_ctc.tar.xz -->
```
Comment (maringeo) [inline assets/docs/tulasiram58827/keras-ocr/dr/1.md:5]: Models on tfhub.dev are immutable, so the correct way would be to create a version 2 of the model. 

To do so you need to create a 2.md with the current changes. 

Also, for better visual organization on tfhub.dev, I suggest to also add a second version (2.md) to the placeholder model (https://raw.githubusercontent.com/tensorflow/tfhub.dev/master/assets/docs/tulasiram58827/keras-ocr/1.md) and then use `<!-- parent-model: tulasiram58827/keras-ocr/2 -->` for the new Lite version.

**Pull request closed without merging**